### PR TITLE
Clean up DNS test

### PR DESCRIPTION
**Pull request by dirkmueller**
Test servers individually rather than all of then at the same
time.

### CODE REVIEW

File: backup/crowbar-backup [hunk 1]
```diff
@@ -88,12 +88,8 @@ function check_resolv_conf
 {
   # check if DNS servers are available and can resolve domain names
   for ns in `grep "^\s*nameserver" /etc/resolv.conf | sed -e "s/\s*nameserver \([[:digit:]\.]\+\)[^[:digit:]\.]*/\1/" ` ; do
-    if ! netcat -zu $ns 53 ; then
-      echo "Error: Nameserver '$ns' is not available. Please reconfigure /etc/resolv.conf manually."
-      exit 1
-    fi
-    if ! getent hosts "$HOSTTEST" >/dev/null; then
-      echo "Error: Nameserver '$ns' can not resolve or ping $HOSTTEST. Please reconfigure /etc/resolv.conf manually"
+    if ! dig @$ns +short "$HOSTTEST" >/dev/null; then
```
Comment (vuntz) [inline backup/crowbar-backup]: Hrm, dig doesn't seem to return a non-0 value when there's no result.

So -1.

Comment (mimi1vx) [inline backup/crowbar-backup]: SLE12
doener3:~ # dig @1.1.1.1 +short www.suse.com
;; connection timed out; no servers could be reached
doener3:~ # echo $?
9
doener3:~ # dig @8.8.8.8 +short www.suse.com                                                                                                                                                                                                                                     
130.57.66.10
doener3:~ # echo $?
0
doener3:~ #

Comment (dirkmueller) [inline backup/crowbar-backup]: > Hrm, dig doesn't seem to return a non-0 value when there's no result.
> So -1.

@vuntz  it doesn't actually matter whether there's a result or not, it only matters if the server responds. if the server responds with "no such host", then we know the server is there. it properly returns a non-zero exit code if the server does not exist or is down.

**Comment by vuntz** [comment]
Oh, ok +1

**Pull request merged**